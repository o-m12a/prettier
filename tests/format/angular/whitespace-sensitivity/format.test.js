runFormatTest(import.meta, ["angular"], { printWidth: 10 });
runFormatTest(import.meta, ["angular"], {
  printWidth: 10,
  htmlWhitespaceSensitivity: "css",
});
runFormatTest(import.meta, ["angular"], {
  printWidth: 10,
  htmlWhitespaceSensitivity: "ignore",
});
