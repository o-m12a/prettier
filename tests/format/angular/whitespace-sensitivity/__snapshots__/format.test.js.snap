// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pure-single-interpolation-only.html - {"printWidth":10,"htmlWhitespaceSensitivity":"css"} format 1`] = `
====================================options=====================================
htmlWhitespaceSensitivity: "css"
parsers: ["angular"]
printWidth: 10
          | printWidth
=====================================input======================================
<h1>{{ foo }}</h1>

<!-- whitespaces -->

<h1>
  {{ foo }}
</h1>

<h1>  {{ foo }}  </h1>

<!-- impurities -->

<h1>leading {{ foo }} trailing</h1>

<!-- multiple interpolations -->

<h1>{{ foo }} {{ bar }}</h1>

=====================================output=====================================
<h1>{{
  foo
}}</h1>

<!-- whitespaces -->

<h1>
  {{
    foo
  }}
</h1>

<h1>
  {{
    foo
  }}
</h1>

<!-- impurities -->

<h1>
  leading
  {{
    foo
  }}
  trailing
</h1>

<!-- multiple interpolations -->

<h1>
  {{
    foo
  }}
  {{
    bar
  }}
</h1>

================================================================================
`;

exports[`pure-single-interpolation-only.html - {"printWidth":10,"htmlWhitespaceSensitivity":"ignore"} format 1`] = `
====================================options=====================================
htmlWhitespaceSensitivity: "ignore"
parsers: ["angular"]
printWidth: 10
          | printWidth
=====================================input======================================
<h1>{{ foo }}</h1>

<!-- whitespaces -->

<h1>
  {{ foo }}
</h1>

<h1>  {{ foo }}  </h1>

<!-- impurities -->

<h1>leading {{ foo }} trailing</h1>

<!-- multiple interpolations -->

<h1>{{ foo }} {{ bar }}</h1>

=====================================output=====================================
<h1>{{
  foo
}}</h1>

<!-- whitespaces -->

<h1>
  {{
    foo
  }}
</h1>

<h1>
  {{
    foo
  }}
</h1>

<!-- impurities -->

<h1>
  leading
  {{
    foo
  }}
  trailing
</h1>

<!-- multiple interpolations -->

<h1>
  {{
    foo
  }}
  {{
    bar
  }}
</h1>

================================================================================
`;

exports[`pure-single-interpolation-only.html - {"printWidth":10} format 1`] = `
====================================options=====================================
parsers: ["angular"]
printWidth: 10
          | printWidth
=====================================input======================================
<h1>{{ foo }}</h1>

<!-- whitespaces -->

<h1>
  {{ foo }}
</h1>

<h1>  {{ foo }}  </h1>

<!-- impurities -->

<h1>leading {{ foo }} trailing</h1>

<!-- multiple interpolations -->

<h1>{{ foo }} {{ bar }}</h1>

=====================================output=====================================
<h1>{{
  foo
}}</h1>

<!-- whitespaces -->

<h1>
  {{
    foo
  }}
</h1>

<h1>
  {{
    foo
  }}
</h1>

<!-- impurities -->

<h1>
  leading
  {{
    foo
  }}
  trailing
</h1>

<!-- multiple interpolations -->

<h1>
  {{
    foo
  }}
  {{
    bar
  }}
</h1>

================================================================================
`;
